recipes/default.rb

```diff
@@ -17,5 +17,4 @@
 # limitations under the License.
 #
 
-include_recipe 'osl-php::packages'
```

Comment:
This needs to remain as its expected to be there when you just include the cookbook normally.